daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/ChoiceGroup.scala

```diff
@@ -78,29 +77,48 @@ import edu.illinois.ncsa.daffodil.api.WarnID
  *
  */
 
-final class Choice(xmlArg: Node, parent: SchemaComponent, position: Int)
-  extends ModelGroup(xmlArg, parent, position)
-  with Choice_AnnotationMixin
-  with RawDelimitedRuntimeValuedPropertiesMixin // initiator and terminator (not separator)
-  with ChoiceGrammarMixin {
+trait ChoiceLikeMixin
```

Comment:
Scaladoc, and/or change trait name to be more descriptive. What aspect of Choices does ChoiceLike capture?